src/errors/test-run/templates.js

```diff
@@ -339,5 +339,17 @@ export default {
 
     [TEST_RUN_ERRORS.childWindowClosedBeforeSwitchingError]: () => `
         The child window was closed before TestCafe could switch to it.
-    `
+    `,
+
+    [TEST_RUN_ERRORS.cannotCloseWindowWithChildrenError]: () => `
+        Cannot close window with opened child windows.
+    `,
+
+    [TEST_RUN_ERRORS.targetWindowNotFoundError]: () => `
+        The window defined in command is not found.
+    `,
+
+    [TEST_RUN_ERRORS.allowMultipleWindowsOptionIsNotSpecifiedError]: err => `
+        To use the "${err.methodName}" specify the "allow-multiple-windows" run option.
```

Comment:
You should activate multi window mode (enable the "allow-multiple-windows" option) to use the "${err.methodName}" method.